<?xml version="1.0"?>
<launch>

  <!-- Load the model parameters onto the parameter server -->
  <rosparam command="load" file="$(find grasp_icp)/config/model_list.yaml" />
  <!-- node name="pose_transformation_client" pkg="pose_transformation" type="pose_transformation_cilent.py" output="screen">    
  </node -->
  <node name="pose_transformation_server" pkg="pose_transformation" type="pose_transformation_server.py" output="screen" args="/catkin_ws/src/grasp_icp/config/"> </node>

</launch>
